test/cert-ceremonies/generate.go

```diff
@@ -73,16 +73,19 @@ func genCert(path string) error {
 }
 
 func main() {
+	logger := cmd.NewLogger(cmd.SyslogConfig{StdoutLevel: 6, SyslogLevel: -1})
+	logger.Info("Logging to stdout/stderr only, not syslog.")
+
 	// If one of the output files already exists, assume this ran once
 	// already for the container and don't re-run.
 	outputFile := "/hierarchy/root-signing-pub-rsa.pem"
 	if loc, err := os.Stat(outputFile); err == nil && loc.Mode().IsRegular() {
 		fmt.Println("skipping certificate generation: already exists")
 		return
 	} else if err == nil && !loc.Mode().IsRegular() {
-		log.Fatalf("statting %q: not a regular file", outputFile)
+		logger.Errf("statting %q: not a regular file", outputFile)
```

Comment:
For consistency with the rest of this file I'd use `cmd.Fail` instead of `logger.Errf`.